includes/Admin/Settings_Screens/Connection.php

```diff
@@ -94,6 +94,22 @@ public function enqueue_assets() {
 	}
 
 
+	/**
+	 * Gets the catalog URL based on catalog ID and business ID.
+	 *
+	 * @param string $catalog_id The catalog ID
+	 * @param string $business_id The business ID (optional)
+	 * @return string The catalog URL
+	 */
+	private function get_catalog_url(string $catalog_id, string $business_id = ''): string 
+	{
+		if ($business_id) {
+			return "https://www.facebook.com/commerce/catalogs/{$catalog_id}/products/?business_id={$business_id}";
+		}
+		return "https://facebook.com/products/catalog/{$catalog_id}";
```

Comment:
Can we just use cta ID?

```suggestion
		return "https://facebook.com/products/catalog/{$catalog_id}";
```